pkg/reconciler/taskrun/taskrun.go

```diff
@@ -528,7 +528,7 @@ func (c *Reconciler) reconcile(ctx context.Context, tr *v1beta1.TaskRun, rtr *re
 	tr.Status.TaskSpec = ts
 
 	if len(tr.Status.TaskSpec.Steps) > 0 {
-		logger.Infof("set taskspec for %s/%s - script: %s", tr.Namespace, tr.Name, tr.Status.TaskSpec.Steps[0].Script)
+		logger.Debugf("set taskspec for %s/%s - script: %s", tr.Namespace, tr.Name, tr.Status.TaskSpec.Steps[0].Script)
```

Comment:
🎉  wow, never noticed scripts are logged as `info` 😞 

Just wondring, why is it hardcoded to index `0`?

Reply:
Uhm, good question - we could also remove it completely, but it felt like it could be useful in debug level